setup.py

```diff
@@ -54,7 +55,7 @@ def load_requirements(*requirements_paths):
     ],
     packages=find_packages(include=['openassessment*'], exclude=['*.test', '*.tests']),
     include_package_data=True,
-    install_requires=load_requirements('requirements/base.txt', "requirements/django.txt"),
+    install_requires=load_requirements('requirements/base.in', "requirements/django.txt"),
```

Comment:
Switching this back to `requirements/base.txt` locally at least allowed me to run tests via tox (although, with many failures, which I don't yet understand).